lib/items.rb

```diff
@@ -1,20 +1,35 @@
+require 'bigdecimal'
+require 'time'
+
 class Item
-  attr_reader :id,
-              :name,
-              :description,
-              :unit_price,
-              :created_at,
-              :updated_at,
-              :merchant_id,
-              :unit_price_to_dollars
+  attr_accessor :id,
+                :name,
+                :description,
+                :unit_price,
+                :created_at,
+                :updated_at,
+                :merchant_id
 
   def initialize(attributes)
-    @id = attributes[:id]
+    @id = attributes[:id].to_i
     @name = attributes[:name]
     @description = attributes[:description]
-    @unit_price = attributes[:unit_price]
+    @unit_price = (BigDecimal(attributes[:unit_price])/100)
     @created_at = attributes[:created_at]
-    @updated_at = attributes[:created_at]
-    @merchant_id = attributes[:merchant_id]
+    @updated_at = attributes[:updated_at]
+    @merchant_id = attributes[:merchant_id].to_i
+  end
+
+  def unit_price_to_dollars
+    @unit_price.to_f
+  end
+
+# talk to instructor about using these helper methods 
+  def created_at_to_time
```

Comment:
New methods created are great, will review later to make sure that it wont break out project later since its out the initialize.